### ISSUE TITLE
Cards in the dashboard are not interactive.

### ISSUE THREAD

**Issue by aashimawadhwa** [PalisadoesFoundation/talawa-admin#573]
**Is your feature request related to a problem? Please describe.**
The cards are not interactive yet, we have functionalities for four cards right now,
we can link the: 

1. event card with the event section from the navbar, 
2. post card to the post section from the navbar, 
3. members card to the people section from the navbar.
4.  blocked users card to Block/Unblock section from the navbar

**Describe the solution you'd like**
Add a on click functionalities and link cards to the menu from the navbar as explained above
![Screenshot 2023-03-08 at 9 12 13 AM](https://user-images.githubusercontent.com/73706697/223616127-f038153f-b8b0-4ce6-b9ac-c6c93a6850e8.png)

**Additional context**
**This is a good first issue and we welcome newcomers and beginners to  look into them**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by aashimawadhwa** [comment]
I would like to take this issue

**Comment by beingnoble03** [comment]
@palisadoes I don't think such issues regarding non-functionality of the talawa-admin, will help talawa-admin to improve as we have a project idea for [redesigning the admin UI](https://docs.talawa.io/docs/internships/internship-ideas/#admin-ui-redesign). So, what's the benefit of solving these issues if in upcoming GSoC program, the code for UI is going to be replaced by some better UI code.

**Comment by palisadoes** [comment]
> @palisadoes I don't think such issues regarding non-functionality of the talawa-admin, will help talawa-admin to improve as we have a project idea for [redesigning the admin UI](https://docs.talawa.io/docs/internships/internship-ideas/#admin-ui-redesign). So, what's the benefit of solving these issues if in upcoming GSoC program, the code for UI is going to be replaced by some better UI code. Obviously, functionality based issues would be fruitful as the code can be reused for them.

I agree. However, we have a very large number of new participants from Outreachy who need some entry level tasks to do as part of their evaluations. We have very few test cases left and fewer bugs are being found each week. Some of these basic tasks will be required in whatever the new design may be. There is a rationale, though it may not seem that way from a GSoC perspective.

**Comment by beingnoble03** [comment]
> > @palisadoes I don't think such issues regarding non-functionality of the talawa-admin, will help talawa-admin to improve as we have a project idea for [redesigning the admin UI](https://docs.talawa.io/docs/internships/internship-ideas/#admin-ui-redesign). So, what's the benefit of solving these issues if in upcoming GSoC program, the code for UI is going to be replaced by some better UI code. Obviously, functionality based issues would be fruitful as the code can be reused for them.
> 
> I agree. However, we have a very large number of new participants from Outreachy who need some entry level tasks to do as part of their evaluations. We have very few test cases left and fewer bugs are being found each week. Some of these basic tasks will be required in whatever the new design may be. There is a rationale, though it may not seem that way from a GSoC perspective.

I guess then it must be assured that these type of issues are only assigned to first time contributors. Right?

**Comment by palisadoes** [comment]
Yes, people more experienced in the code base will want greater challenges and are finding them.

**Comment by techfussion** [comment]
Hello I would like to work on this

**Comment by beingnoble03** [comment]
@palisadoes Can we add an `outreachy` tag for such issues? This will help new contributors to find these issues easily.

**Comment by Yass32** [comment]
Hello im a new outreachy applicant. can I be assigned this task

**Comment by Shugo52** [comment]
Hello I'm an outreachy applicant, can I be assigned this task.

**Comment by satyamksharma** [comment]
Hi there, I'm an Outreachy contributor and I would like to contribute to this issue. Can you assign me

**Comment by palisadoes** [comment]
> Hi there, I'm an Outreachy contributor and I would like to contribute to this issue. Can you assign me

If the issue has already been assigned, please don't ask to be assigned. We want everyone to get a chance.

**Comment by palisadoes** [comment]
> Hello I'm an outreachy applicant, can I be assigned this task.

If the issue has already been assigned, please don't ask to be assigned. We want everyone to get a chance.

**Comment by Yass32** [comment]
hey @techfussion can we collaborate on this

**Comment by palisadoes** [comment]
> hey @techfussion can we collaborate on this

Only one issue per person as outlined in our CONTRIBUTING.md file.  If the currently assigned person, or a mentor feels that it is a big enough issue to be split, then that could be done. This is very important if you are planning on applying for an internship. With shared issues, it is difficult to know the level of contribution of each person.

**Comment by palisadoes** [comment]
> hey @techfussion can we collaborate on this

Only one issue per person as outlined in our CONTRIBUTING.md file.  If the currently assigned person, or a mentor feels that it is a big enough issue to be split, then that could be done. This is very important if you are planning on applying for an internship. With shared issues, it is difficult to know the level of contribution of each person.

**Comment by palisadoes** [comment]
Unassigning due to inactivity others need to get a chance. Please let us know whether you want to be reassigned before someone else requests assignment or select another issue.

**Comment by KrutikaBhatt** [comment]
@palisadoes, can I work on this ?

**Comment by techfussion** [comment]
@palisadoes sorry for the inactivity. This issue has already been fixed and merged and is supposed to close automatically. Here's a link to the merged pull request:
https://github.com/PalisadoesFoundation/talawa-admin/pull/620

**Comment by KrutikaBhatt** [comment]
Hey @techfussion, I have mentioned in my PR about merged pull request #620. There are some test case failures around it which is not exactly caused due the file. I will try to cover it though in the PR

**Comment by KrutikaBhatt** [comment]
@palisadoes,the main functionality is already developed and merged. Rest failed test cases caused should be created as separate Issues since its not caused because of it. Sorry for inactivity
